bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-multimocks",
-  "version": "0.5.7",
+  "version": "0.5.8",
```

Comment:
Bower doesn't use this, its uses the github tags for versioning - you can just remove it and theres one less thing to do when committing